reporter/html.js

```diff
@@ -782,9 +782,11 @@ export function escapeText( s ) {
 					escapeText( actual ) + "</pre></td></tr>";
 
 				if ( typeof details.actual === "number" && typeof details.expected === "number" ) {
-					showDiff = true;
-					diff = ( details.actual - details.expected );
-					diff = ( diff > 0 ? "+" : "" ) + diff;
+					if ( !isNaN( details.actual ) && !isNaN( details.expected ) ) {
```

Comment:
I know we are protecting this line with the type check above, but the global `isNaN` is still [broken and unreliable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN). It makes me want to protect the code from further weirdness.

`Number.isNaN` is doing the right thing. Unfortunately, it is not available in IE9, which we support.

If can use a polyfill...

```js
Number.isNaN = Number.isNaN || function(value) {     
    return value !== value;
}
```

or just:

```js
// Confirm the values are not NaN
if ( details.actual === details.actual && details.expected === details.expected ) {
```

I can confirm JS won't have any other value that returns false in a strict comparison, it would break the web, actually. So that is reliable.